app/views/devise/invitations/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :title, t(".title")  %>
```

Comment:
I think that the code in the application layout already add a title if present in the translation model without the need of this code

Reply:
Yeah, I tried that first but it didn't seem to work with the Devise bits. I'll stop being lazy and try a bit harder to make it work 😛

Reply:
Oh! That's odd! If it is going to get a painful bit, don't worry! I am happy to approve this if we decide that is too much hassle :D

Reply:
@cintamani Worked it out - I had forgotten that since we had our own UserInvitationsController it would need a different name in the YAML.